src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -135,9 +135,10 @@ object HttpGoogleServicesDAO extends GoogleServicesDAO with FireCloudRequestBuil
       .refreshAccessToken().getTokenValue
   }
 
+  @deprecated("currently unused and targeted for deletion", "2020-03-06")
   override def getAdminIdentityToken: String = {
     getScopedServiceAccountCredentials(firecloudAdminSACreds, Seq("openid", "email"))
-      .idTokenWithAudience(EntityClient.avroToRawlsURL, List()).getTokenValue
```

Comment:
Is it necessary to deprecate this - can we just delete it if its not used anywhere?